mod-kb-ebsco-java/mod-kb-ebsco-java.postman_collection.json

```diff
@@ -467,14 +470,90 @@
+											"      method: 'PUT',",
+											"      header: {",
+											"        'Content-Type' : 'application/vnd.api+json',",
+											"        'X-Okapi-Tenant' : pm.variables.get(\"xokapitenant\"),",
+											"        'X-Okapi-Token' : pm.variables.get(\"xokapitoken\")",
+											"          ",
+											"      },",
+											"      body: {",
+											"        mode: 'application/vnd.api+json',",
+											"        raw: JSON.stringify(requestBody)",
+											"      }",
+											"    };",
+											"    ",
+											"    pm.sendRequest(credentialsPutRequest, function (err, res) {",
+											"      if (err) { console.log(err); }",
+											"      pm.test('Response must be valid', function () {",
+											"        pm.expect(err).to.equal(null);",
+											"        pm.expect(res).to.have.property('code', 204);",
+											"        pm.environment.set(\"kb-credentials-id1\", credentials.id);",
+											"      });",
+											"    });",
+											"} else {",
+											"",
+											"    var credentials = {",
+											"        type: 'kbCredentials',",
+											"        attributes: {",
+											"            name: 'University of Massachusettss',",
+											"            apiKey: pm.variables.get(\"rm.apiKey\"),",
+											"            url: pm.variables.get(\"rm.apiUrl\"),",
+											"            customerId: pm.variables.get(\"rm.apiCustomer\")",
+											"        }",
+											"    }",
+											"",
+											"    var requestBody = {",
+											"        data: credentials",
+											"    };",
+											"",
+											"    const credentialsPostRequest = {",
+											"      url:  pm.variables.get(\"protocol\") + \"://\" +  pm.variables.get(\"url\") + \":\" + pm.variables.get(\"okapiport\") + '/eholdings/kb-credentials',",
+											"      method: 'POST',",
+											"      header: {",
+											"        'Content-Type' : 'application/vnd.api+json',",
+											"        'X-Okapi-Tenant' : pm.variables.get(\"xokapitenant\"),",
+											"        'X-Okapi-Token' : pm.variables.get(\"xokapitoken\")",
+											"          ",
+											"      },",
+											"      body: {",
+											"        mode: 'application/vnd.api+json',",
+											"        raw: JSON.stringify(requestBody)",
+											"      }",
+											"    };",
+											"    ",
+											"    pm.sendRequest(credentialsPostRequest, function (err, res) {",
+											"      if (err) { console.log(err); }",
+											"      pm.test('Response must be valid', function () {",
+											"        pm.expect(err).to.equal(null);",
+											"        pm.expect(res).to.have.property('code', 201);",
+											"        pm.environment.set(\"kb-credentials-id1\", res.json().id);",
+											"      });",
+											"    });",
+											"}",
+											"",
 											"pm.environment.set(\"kb-credentials-id1\", jsonData.id);"
```

Comment:
can be removed, because you set an environment variable for both cases.

Reply:
fixed